common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -252,6 +260,7 @@ def student_view(self, context):
             'next_url': context.get('next_url'),
             'prev_url': context.get('prev_url'),
             'override_hidden_exam': masquerading and special_exam_html is not None,
+            'gated_banner': content_milestones and self.runtime.user_is_staff,
```

Comment:
I might have understood the ticket incorrectly. I thought @sanfordstudent ticket was to allow staff to view gated content in the lms as staff. When staff does this, we were supposed to inform staff that the content is gated by displaying this banner.